stdlib/runtime/numbers.gr

```diff
@@ -1240,17 +1241,17 @@ export let coerceNumberToInt64 = (x: Number) => {
 }
 
 export let coerceNumberToRational = (x: Number) => {
-  let x = WasmI32.fromGrain(x)
-  let result = if (isSimpleNumber(x)) {
-    newRational(untagSimple(x), 1n)
+  let y = WasmI32.fromGrain(x)
+  let result = if (isSimpleNumber(y)) {
+    newRational(untagSimple(y), 1n)
   } else {
-    let tag = boxedNumberTag(x)
+    let tag = boxedNumberTag(y)
     if (WasmI32.eq(tag, Tags._GRAIN_RATIONAL_BOXED_NUM_TAG)) {
-      x
+      y
     } else if (WasmI32.eq(tag, Tags._GRAIN_INT32_BOXED_NUM_TAG)) {
-      newRational(boxedInt32Number(x), 1n)
+      newRational(boxedInt32Number(y), 1n)
     } else if (WasmI32.eq(tag, Tags._GRAIN_INT64_BOXED_NUM_TAG)) {
-      newRational(coerceInt32(x), 1n)
+      newRational(coerceNumberToWasmI32(x), 1n)
```

Comment:
I think it would have been fine to leave the variable name as `x` and cast `x` back to a Number here:
```suggestion
      newRational(coerceNumberToWasmI32(WasmI32.toGrain(x) : Number, 1n)
```

Reply:
Changed back to x as suggested.